numbat-cli/src/main.rs

```diff
@@ -284,6 +285,19 @@ impl Cli {
                             "quit" | "exit" => {
                                 return Ok(());
                             }
+                            "help" | "?" => {
+                                let pretty_print = match self.args.pretty_print {
+                                    PrettyPrintMode::Always => true,
+                                    PrettyPrintMode::Never => false,
+                                    PrettyPrintMode::Auto => true,
+                                };
```

Comment:
I would switch off pretty printing regardless of user settings for these help examples. Makes the help text more concise. What do you think?

Reply:
Yea that makes sense, I can put that into the `help_markup` function as well to help make it more concise.